jiracli/password.go

```diff
@@ -43,6 +50,21 @@ func (o *GlobalOptions) GetPass() string {
 			if err != nil {
 				panic(err)
 			}
+		} else if o.PasswordSource.Value == "gopass" {
+			if o.PasswordDirectory.Value != "" {
+				orig := os.Getenv("PASSWORD_STORE_DIR")
+				os.Setenv("PASSWORD_STORE_DIR", o.PasswordDirectory.Value)
+				defer os.Setenv("PASSWORD_STORE_DIR", orig)
+			}
+			if bin, err := exec.LookPath("gopass"); err == nil {
+				buf := bytes.NewBufferString("")
+				cmd := exec.Command(bin, "show", "-o", o.keyName())
+				cmd.Stdout = buf
+				cmd.Stderr = buf
+				if err := cmd.Run(); err == nil {
```

Comment:
Could you add a failure case too please, panic(err) is fine.

Reply:
I just noticed, that the pass password-source doesn't have error handling at all. Should i add it there too?

Reply:
That'd be lovely, thanks!

Reply:
should be done